ui/src/message_popup/popup_question.jsx

```diff
@@ -148,7 +148,6 @@ export default React.createClass({
     return (
       <div>
         <VelocityTransitionGroup leave={{animation: "slideUp"}} runOnMount={true}>
-          {!scaffolding.shouldShowSummary && this.renderQuestion()}
           {!isDoneRevising && scaffolding.shouldShowSummary && this.renderQuestion()}
```

Comment:
ah thanks, great review! 👍 I'll drop this then and am looking at refactoring some of this for audio responses and will clarify separately.